README.md

```diff
@@ -1,7 +1,100 @@
-Generate a compatibility matrix for plugins against Jenkins core.
+Plugin Compatibility Tester (PCT)
+------
+
+Generates a compatibility matrix for plugins against Jenkins core.
 
 See https://wiki.jenkins-ci.org/display/JENKINS/Plugin+Compatibility+Tester for background.
 
+## Running PCT 
+
+### Running PCT in Docker
+
+It is recommended to run PCT in the prepared Docker image.
+You can find it [here](hub.docker.com/r/jenkins/pct/).
+
+#### Examples
+
+This command will clone of a repository and run PCT against the latest Jenkins Core:
+
+```shell
+docker run --rm -v maven-repo:/root/.m2 -v $(pwd)/out:/pct/out -e ARTIFACT_ID=ssh-slaves -e VERSION=ssh-slaves-1.24 jenkins/pct
+```
+
+This command will run PCT for the latest plugin version against the specified Jenkins WAR.
+PCT supports running against SNAPSHOT builds, but PCT will have to install local Maven artifacts in such case.
+
+```shell
+docker run --rm -v maven-repo:/root/.m2 -v $(pwd)/out:/pct/out -v my/jenkins.war:/pct/jenkins.war:ro -e ARTIFACT_ID=ssh-slaves jenkins/pct
```

Comment:
Rather than `$(pwd)/out` I would just recommend using `/tmp/pct` or the like, and then we can drop the `.gitignore` entry.